aws-environment-param-sync/src/index.js

```diff
@@ -0,0 +1,177 @@
+const core = require('@actions/core');
+const { SSM } = require('aws-sdk');
+const { Octokit } = require("@octokit/core");
+
+const getParameter = async ({
+     path,
+  }) => {
+  const ssm = new SSM();
+  const parameters = {};
```

Comment:
For the future, [javascript map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) is a better suited data structure for key value mappings